### PR TITLE
Better PosAnimation example

### PR DESCRIPTION
**Pull request by stell**
Fixes https://github.com/Leaflet/Leaflet/issues/6175.

### CODE REVIEW

File: src/dom/PosAnimation.js [hunk 1]
```diff
@@ -11,8 +11,22 @@ import * as DomUtil from '../dom/DomUtil';
  *
  * @example
  * ```js
- * var fx = new L.PosAnimation();
- * fx.run(el, [300, 500], 0.5);
+ * var myPositionMarker = L.marker([48.864716, 2.294694]).addTo(map);
+ * 
+ * myPositionMarker.on("click", function(){
```
Comment (rheh) [inline src/dom/PosAnimation.js:16]: For consistency add a space between () and {?

File: src/dom/PosAnimation.js [hunk 2]
```diff
@@ -11,8 +11,22 @@ import * as DomUtil from '../dom/DomUtil';
...
+ * 	var pos = map.latLngToLayerPoint(myPositionMarker.getLatLng())
```
Comment (rheh) [inline src/dom/PosAnimation.js:17]: No semi-colon on end of line.  Should we add one to be consistant?

File: src/dom/PosAnimation.js [hunk 3]
```diff
@@ -11,8 +11,22 @@ import * as DomUtil from '../dom/DomUtil';
...
+ * 	pos.y -= 25;
+ * 	var fx = new L.PosAnimation();
+ * 
+ * 	fx.once('end',function(){
```
Comment (rheh) [inline src/dom/PosAnimation.js:21]: Again space between () and {

File: src/dom/PosAnimation.js [hunk 4]
```diff
@@ -11,8 +11,22 @@ import * as DomUtil from '../dom/DomUtil';
...
+ * 		pos.y += 25;
+ * 		fx.run(myPositionMarker._icon, pos, 0.8);
+ * 	});
+ * 
+ * 	fx.run(myPositionMarker._icon, pos, 0.3);
+ *
```
Comment (rheh) [inline src/dom/PosAnimation.js:27]: Remove blank line?

**Review by rheh — CHANGES_REQUESTED** [review]

**Review by rheh — APPROVED** [review]

**Comment by johnd0e** [comment]
@stell @rheh 
Haven't you noticed failed CI-checks?

**Review by samuei — COMMENTED** [review]
Failed CI messages are from trailing spaces on lines 15, 20, 25, and 28.

**Comment by rheh** [comment]
> @stell @rheh
> Haven't you noticed failed CI-checks?

TBH I did not, I just reviewed the code.

**Review by Falke-Design — APPROVED** [review]

**Comment by Falke-Design** [comment]
@Malvoz can you please review

**Review by Malvoz — APPROVED** [review]

**Pull request merged**